### PR TITLE
Update projects.json adding MLNET

### PR DESCRIPTION
**Pull request by shauheen**

### CODE REVIEW

File: projects/projects.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
 "contributor": "Microsoft"},
 {"name":"aspnet-core.md",
 "contributor": "Microsoft"},
+{"name":"mlnet.md",
```
Comment (danmosemsft) [inline projects/projects.json]: Did you add mlnet.md?

Comment (shauheen) [reply projects/projects.json]: yes, it is in PR #101

**Comment by davidfowl** [comment]
project.json.... bad name

**Comment by shauheen** [comment]
@davidfowl would you please explain, bad name for what?

**Comment by davidfowl** [comment]
It's a historical .NET thing 😄

**Comment by Petermarcu** [comment]
@davidfowl you are right that it has a bad resemblance to the project type that shall not be named. That said, this PR isn't renaming it and this is the plural "projects.json". :)

**Comment by davidfowl** [comment]
Since I'm here, I think we should reformat the JSON

**Comment by Petermarcu** [comment]
To get some whitespace in there?

**Comment by jkotas** [comment]
> this PR isn't renaming it and this is the plural "projects.json"

And other home-like repos use the same name and similar structure too. One example from many: https://github.com/angular/builtwith.angularjs.org/blob/master/projects/projects.json

**Pull request merged**